### ISSUE TITLE
Dockerhub or some repo

### ISSUE THREAD

**Issue by mels0n** [Shadowghost/jellyfin-rffmpeg#6]
Hi is this building to dockerhub or some repo somewhere?

**Comment by mels0n** [comment]
If anyone else stumbles across this. I'm pretty confident this is the same container.

https://hub.docker.com/r/bitwrk/jellyfin-rffmpeg

**Comment by Shadowghost** [comment]
Yes it is. The builds are automated and  I'm currently not using rffmpeg anymore so if anything does not work, please let me know.
